src/commands/Misc/remindme.ts

```diff
@@ -37,7 +37,7 @@ export class UserCommand extends SkyraCommand {
 	}
 
 	public async create(message: Message, args: SkyraCommand.Args) {
-		const duration = await args.pick('timespan', { minimum: minutes(1) });
+		const duration = await args.pick('timespan', { minimum: minutes(1), maximum: years(1) });
```

Comment:
```suggestion
		const duration = await args.pick('timespan', { minimum: minutes(1), maximum: years(200) });
```

1 year is very short. Lets allow at least 200.

Reply:
That's too long, let's make it 5.
```suggestion
		const duration = await args.pick('timespan', { minimum: minutes(1), maximum: years(5) });
```

Reply:
Why is 200 too long? 200 years from now is 2222, not 10_000.